dist/common/scripts/scylla_raid_setup

```diff
@@ -88,10 +88,17 @@ if __name__ == '__main__':
 
     if is_debian_variant():
         run('env DEBIAN_FRONTEND=noninteractive apt-get -y install mdadm xfsprogs')
+        md_service = 'mdadm.service'
+        try:
+            mdmonitor = systemd_unit('mdmonitor.service')
```

Comment:
Shouldn't you update md_service here?

Reply:
Sorry previous code were wrong, pushed new version. please review again